app/src/main/java/com/openclassrooms/realestatemanager/ui/EstateEditFragment.kt

```diff
@@ -166,6 +179,24 @@ class EstateEditFragment(
         countryEditText.setText(estate.country)
     }
 
+    private val clickOnEntryDate = OnClickListener {
+        val immutableEntryDate = entryDate
+        val entryDateCalendar = Calendar.getInstance()
+        immutableEntryDate?.let { entryDateCalendar.timeInMillis = immutableEntryDate }
```

Comment:
`.let{ it -> ....}`  te renvoie un `it` immuable.

Tu peux donc simplifer
```suggestion
        val entryDateCalendar = Calendar.getInstance()
        entryDate?.let { entryDateCalendar.timeInMillis = it }
```

Reply:
Ok merci